bundles/persistence/org.openhab.persistence.rrd4j/src/main/java/org/openhab/persistence/rrd4j/internal/RRD4jService.java

```diff
@@ -307,6 +309,10 @@ private State mapToState(double value, String itemName) {
 					return value==0.0d ? OnOffType.OFF : OnOffType.ON;
 				} else if(item instanceof ContactItem) {
 					return value==0.0d ? OpenClosedType.CLOSED : OpenClosedType.OPEN;
+				} else if(item instanceof DimmerItem || item instanceof RollershutterItem) {
+					// make sure Items that need PercentTypes instead of DecimalTypes
+					// do receive the right information
+					return new PercentType( (int) Math.round(value * 100) );
```

Comment:
It seems the getStateAs method of DimmerItem and RollershutterItem do provide the state value to the persistence service divided by 100. Because of this the bug mentioned in issue #3090 is happening. 
If the rollershutter is on 70% while it was stored. It is just 0.7 in the database.

Changing the getStateAs methods seems to risky to me, has this might create impacts to existing rules and scripts which might use this methods.